src/VTMap.js

```diff
@@ -0,0 +1,371 @@
+import React from 'react';
+import './App.css';
+import { Map, TileLayer, Polygon, Polyline } from 'react-leaflet';
+import borderData from './border.js';
+import L from 'leaflet'
+import leafletPip from 'leaflet-pip'
+let defaultZoom = 7
+
+class VTMap extends React.Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            latitude: '',
+            longitude: '',
+            county: '',
+            town: '',
+            status: 'Game Started, Awaiting your first guess..',
+            guess: '',
+            modalDisplayed: false,
+            points: 100,
+            mapLat: 44.0886,
+            mapLon: -72.7317,
+            locationArray: [],
+            highScore: '',
+            username: 'Unknown'
```

Comment:
You're close to nailing that highScore! Give the user a chance to enter name (by way of state), and display it using the methods tied to `localStorage` found [here](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)